packages/angular_devkit/build_angular/src/utils/action-executor.ts

```diff
@@ -62,6 +63,7 @@ export class BundleActionExecutor {
     return (this.largeWorker = new JestWorker(workerFile, {
       exposedMethods: ['process', 'inlineLocales'],
       setupArgs: [[...serialize(this.workerOptions)]],
+      numWorkers: maxWorkers,
```

Comment:
I'm not sure how this interacts with the `numWorkers` below (for `this.smallWorker`). I imagine its additive.

Other than that it LGTM, but would like @clydin's input.